### DISCUSSION TITLE
Enshrouded - no GPU found

### DISCUSSION THREAD

**Discussion by ChrischiHam** [Whisky-App/Whisky#852] [Q&A]
Enshrouded is not starting due to missing compatible GPU driver, any help?
![Bildschirmfoto 2024-02-29 um 18 48 44](https://github.com/Whisky-App/Whisky/assets/2826693/d0c536ea-2722-4d32-aa2c-37a84ce807e1)

**Comment by ChrischiHam** [comment]
[E 00:00:04,414] [graphics] No usable Vulkan device found!
[E 00:00:04,414] [graphics] Could not create vulkan device! error=not found
[E 00:00:04,414] [graphics] Could not initialize graphics device!
[X 00:00:04,415] FATAL ERROR: Graphics_NoCompatibleDeviceFound

**Reply by bolt325** [reply]
found any solution?

**Reply by Max094Reikeb** [reply]
I am having the same issue... please would anyone know how to fix this?

**Comment by yllekz** [comment]
I've run into this trying to get the Windows 7 card games to work on macOS (they work for me on Linux with ProtonGE as a runner in Bottles) - I think it's the lack of Vulkan/OpenGL graphics that is making this difficult. I don't know the fix.

<img width="354" alt="image" src="https://github.com/user-attachments/assets/6bdc0486-8d04-4923-95c4-857c761623b6" />

**Comment by ChrischiHam** [comment]
Not yet, but I think that enshrouded requests at gamestart the hardware components, maybe it possible to deactivate by advanced start settings in steam this query. But I don't know the commands.
